.circleci/config.yml

```diff
@@ -210,6 +210,10 @@ jobs:
       release:
         description: the release name (stretch|buster|bionic|focal)
         type: string
+      check:
```

Comment:
To avoid confusion, maybe we should rename the job from 'distcheck' to just 'build', and call the param 'buildtarget', and edit the 'name' below from "Extract and distcheck" to "Build (<< param.buildtarget >>)"